### PR TITLE
Updated API definitions for certification

### PR DESCRIPTION
**Pull request by NicolasValenzuela**
Files have been updated to versions currently in the certification process

---
Please check the following conditions for your PR.

- [x] `apiDefinition.swagger.json` is validated using `paconn validate` command.
- [x] `apiProperties.json` has a valid brand color. Invalid brand colors are `#007ee5` and `#ffffff`.

### CODE REVIEW

**Comment by sriyen-msft** [comment]
@NicolasValenzuela Can you please move connector from "custom-connectors" to "certified-connectors" folder and resolve the branch conflicts in the PR.

**Comment by dougelliott** [comment]
@NicolasValenzuela Checking in - would you be able to address the items that Srikanth outlined in his comment?

**Comment by NicolasValenzuela** [comment]
Hi,

I will review Srikanth's comment.

Thanks doug


Nicolás Valenzuela Guerra
Ingeniero Senior
[cid:image001.png@01D6E632.B9A3C780]

Mail: nvalenzuela@latinshare.com<mailto:nvalenzuela@latinshare.com>
Mobile: 56 9 93478458
www.latinshare.com<http://www.latinshare.cl/>












De: Doug Elliott <notifications@github.com>
Enviado el: viernes, 8 de enero de 2021 22:31
Para: microsoft/PowerPlatformConnectors <PowerPlatformConnectors@noreply.github.com>
CC: Requiem <n.valenzuelag@hotmail.com>; Mention <mention@noreply.github.com>
Asunto: Re: [microsoft/PowerPlatformConnectors] Updated API definitions for certification (#595)


@NicolasValenzuela<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FNicolasValenzuela&data=04%7C01%7Cnvalenzuela%40latinshare.com%7Ca743f16c64204120bb3d08d8b43e38f1%7C530c754d8e524922bcb94da487f8d475%7C0%7C0%7C637457526684763377%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=A9rKQ0RuRYg6uCspwieGyO8VT0aC3RE4Q%2FdBt0nZRQ8%3D&reserved=0> Checking in - would you be able to address the items that Srikanth outlined in his comment?

-

**Comment by NicolasValenzuela** [comment]
@sriyen-msft  @dougelliott 

- LatinShare connectors were moved from the custom connectors folder to certified connectors.
- The LatinShare connectors have been removed from the custom connectors folder.
- The connector has been verified to have a valid marking color.
- The paconn validate command has validated the updated version.
- The version of the file has been modified, based on the version provided by the certification site.

File: certified-connectors/LatinShare SHP Management/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,1060 @@
+{
```
Comment (sriyen-msft) [inline certified-connectors/LatinShare SHP Management/apiDefinition.swagger.json:1]: @NicolasValenzuela This swagger file is not up to date, can you please update it with the latest version.

File: certified-connectors/LatinShare SHP Permissions/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,505 @@
+{
```
Comment (sriyen-msft) [inline certified-connectors/LatinShare SHP Permissions/apiDefinition.swagger.json:1]: @NicolasValenzuela This swagger file is not up to date, can you please update it with the latest version.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/LatinShare SHP Permissions/apiDefinition.swagger.json (hunk 2 above)
Comment (NicolasValenzuela) [reply certified-connectors/LatinShare SHP Permissions/apiDefinition.swagger.json:1]: Okay, it's up to date with the latest version. I'll let you know when it's ready

**Comment by NicolasValenzuela** [comment]
The pull request has been closed, in order to create a new one where the files are updated with the latest version.

**Pull request closed without merging**